libs/cli/langgraph_cli/config.py

```diff
@@ -86,6 +86,33 @@ class AuthConfig(TypedDict, total=False):
     """
 
 
+class CorsConfig(TypedDict, total=False):
+    allow_origins: list[str]
+    allow_methods: list[str]
+    allow_headers: list[str]
+    allow_credentials: bool
+    allow_origin_regex: str
+    expose_headers: list[str]
+    max_age: int
+
+
+class HttpConfig(TypedDict, total=False):
```

Comment:
is `httpconfig` the common name?

Reply:
haha i was debating this with nuno. Unsure. It has:
- Custom app (with custom routes, middleware, and lifespan)
- route toggling
- CORS
could add port configuraiton, etc.

Reply:
maybe something in the style of:

AppConfig
ServiceConfig


---

I don't see the custom routes, middleware or lifespan right -- plan is to add to this very same container?

Reply:
that's all defined in the custom app (via the app path)

Reply:
App doesn't mean anything. Service will be ambiguous when/if we add non http transports. I'd stick with http